src/components/HorizontalTab/index.tsx

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line no-use-before-define
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The ESLint rule `no-use-before-define` is being disabled here. Disabling linting rules can lead to code that's harder to maintain and debug. If possible, try to refactor your code to avoid disabling this rule.



<!-- This is an auto-generated comment by OSS CodeRabbit -->